sparknacl.gpr

```diff
@@ -0,0 +1,138 @@
+library project Sparknacl is
+
+   for Library_Name use "Sparknacl";
+   for Library_Version use "0.0.0";
+
+   for Source_Dirs use ("src");
+   for Object_Dir use "obj";
+   for Create_Missing_Dirs use "True";
+   for Library_Dir use "lib";
+
+   type Library_Type_Type is ("relocatable", "static", "static-pic");
+   Library_Type : Library_Type_Type :=
+     external ("SPARKNACL_LIBRARY_TYPE", external ("LIBRARY_TYPE", "static"));
+   for Library_Kind use Library_Type;
+
+   type Enabled_Kind is ("enabled", "disabled");
+   Compile_Checks : Enabled_Kind := External ("SPARKNACL_COMPILE_CHECKS", "enabled");
```

Comment:
One more thing, for integration in Alire it is best to have the most performance by default, so disable all the checks. In particular for a SPARK project.

Reply:
OK... I will switch default build to no checks, no contracts and -O2 which offers the best size/performance tradeoff. (O3 is a tiny bit faster but results in *much* bigger code size owing to loop unrolling.)